test/optimize/test_optimize.py

```diff
@@ -258,7 +258,7 @@ def get_optimizer(library, solver):
     elif library == 'fides':
         options[fides.Options.SUBSPACE_DIM] = solver[1]
         optimizer = optimize.FidesOptimizer(
-            options=options, hessian_update=solver[0], verbose=0
+            options=options, hessian_update=solver[0], verbose=40
```

Comment:
I guess we'll want `verbose=logging.WARNING` or `logging.ERROR` or whatever 40 is.

Reply:
yes I think we changed it at the same time :D

Reply:
ah, I meant literally writing the logging.XYZ, which is just an int of value 40, but improves readability.